src/Concerns/RegistersEventListeners.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Appkeep\Laravel\Concerns;
+
+use Illuminate\Support\Facades\Event;
+
+trait RegisterEventListeners
+{
+    protected function registerEventListeners()
+    {
+        Event::listen(
+            QueryExecuted::class,
+            [QueryExecutedListener::class, 'handle']
+        );
+
+        Event::listen(
+            SlowQueryEvent::class,
+            [SlowQueryListener::class, 'handle']
```

Comment:
Can we use `DB::whenQueryingForLongerThan` here? That means this feature will only be available for Laravel 9+, but I think that's ok, given that we don't have to worry about the underlying implementation of this listener ourselves.

More info:
https://laravel.com/docs/9.x/database#monitoring-cumulative-query-time